typescript/src/entity.ts

```diff
@@ -359,9 +364,6 @@ export class Entity {
     if (options && options.includeArchived) {
       fetchOptions.query.push(['include_archived', 'true']);
```

Comment:
now the backend support includedArchived to be a json now, but sdk only push the 'true' value